src/layouts/default.html.eco

```diff
@@ -64,7 +64,7 @@
 					<% end %>
 					<li>
 						<a href="//discourse.transformap.co">
-							Forum
+							Agora
```

Comment:
I'm using my veto against Forum. Maybe we should escalate this question to Loomio or DemocracyOS.